plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -0,0 +1,181 @@
+			),
+
+			array(
+				'id'          => 'ai_describe_store_description',
+				'type'        => 'textarea',
+				'title'       => __( 'Describe your store', 'woocommerce' ),
+				'desc_tip'    => __( 'The more detail you provide, the better job our AI can do! Try to include what makes your business unique, who your audience is, and how many products you plan to display.', 'woocommerce' ),
+				'placeholder' => __( 'e.g. Marianne Renoir is a greengrocery taken over by a ten generations Parisian family who wants to keep quality and tradition in the quarter of Montmartre', 'woocommerce' ),
+				'css'         => 'min-width:300px;min-height: 130px;',
+			),
+
+			array(
+				'type' => 'sectionend',
+				'id'   => 'ai_describe_store',
+			),
+
+			array(
+				'title' => __( 'Tone of voice', 'woocommerce' ),
+				'type'  => 'title',
+				'desc'  => __( "Choose the tone of voice you'd like to use to communicate with your customers.", 'woocommerce' ),
+				'id'    => 'ai_tone_of_voice',
+			),
+
+			array(
+				'title'    => __( 'Tone', 'woocommerce' ),
+				'id'       => 'ai_tone_of_voice_select',
+				'class'    => 'wc-enhanced-select',
+				'css'      => 'min-width:300px;',
+				'default'  => 'informal',
+				'type'     => 'select',
+				'options'  => array(
+					'informal'     => __( 'Informal', 'woocommerce' ),
+					'humorous'     => __( 'Humorous', 'woocommerce' ),
+					'neutral'      => __( 'Neutral', 'woocommerce' ),
+					'youthful'     => __( 'Youthful', 'woocommerce' ),
+					'formal'       => __( 'Formal', 'woocommerce' ),
+					'motivational' => __( 'Motivational', 'woocommerce' ),
+				),
+				'desc_tip' => true,
+			),
+
+			array(
+				'type' => 'sectionend',
+				'id'   => 'ai_tone_of_voice',
+			),
+
+			array(
+				'title' => __( 'Store style', 'woocommerce' ),
+				'type'  => 'title',
+				'desc'  => __( "Select the look and feel that best fits your brand - we'll base the layout, color palette, and fonts used on your store around this.", 'woocommerce' ),
+				'id'    => 'ai_store_style',
+			),
+
+			array(
+				'title'    => __( 'Style', 'woocommerce' ),
+				'id'       => 'ai_store_style_select',
+				'class'    => 'wc-enhanced-select',
+				'css'      => 'min-width:300px;',
+				'default'  => 'aa',
+				'type'     => 'select',
+				'options'  => array(
+					'modern'    => __( 'Modern', 'woocommerce' ),
+					'bold'      => __( 'Bold', 'woocommerce' ),
+					'classic'   => __( 'Classic', 'woocommerce' ),
+					'energetic' => __( 'Energetic', 'woocommerce' ),
+					'retro'     => __( 'Retro', 'woocommerce' ),
+					'warm'      => __( 'Warm', 'woocommerce' ),
+				),
+				'desc_tip' => true,
+			),
+
+			array(
+				'type' => 'sectionend',
+				'id'   => 'ai_store_style',
+			),
+		);
+	}
+
+	/**
+	 * Add this page to settings.
+	 *
+	 * @param array $settings_tabs Existing pages.
+	 * @return array|mixed
+	 */
+	public function add_woo_ai_settings_tab( $settings_tabs ) {
+		$settings_tabs[ $this->id ] = __( 'AI', 'woocommerce' );
```

Comment:
We [chatted about this in Slack](p1687553498590979/1687550267.272139-slack-C056QSJJQ91) but I think I'm a fan of moving this to within Advanced as you mentioned:
`Settings -> Advanced -> AI`

Reply:
Yup, done in https://github.com/woocommerce/woocommerce/pull/38920/commits/596bc8231c43063f7be0267ed276483beb07e563.